src/Cli/dotnet/commands/dotnet-test/MSBuildUtility.cs

```diff
@@ -25,6 +25,9 @@ public static (IEnumerable<TestModule> Projects, bool IsBuiltOrRestored) GetProj
                     Path.GetDirectoryName(solutionModel.Description) :
                     SolutionAndProjectUtility.GetRootDirectory(solutionFilePath);
 
+            // TODO: We should pass a binary logger if the dotnet test invocation passed one.
+            // We will take the same file name but append something to it, like `-dotnet-test-evaluation`
+            // Tracked by https://github.com/dotnet/sdk/issues/47494
```

Comment:
We should consider reusing/extracting the logic that `run` has for doing this same task: https://github.com/dotnet/sdk/blob/21a79a39c0d506c83c458238cf6ea9b0e5df3fec/src/Cli/dotnet/commands/dotnet-run/RunCommand.cs#L376-L413

Reply:
This TODO will be handled in a separate PR. I copied your comment in https://github.com/dotnet/sdk/issues/47494#issuecomment-2724022930